pkg/jobrunaggregator/jobrunaggregatoranalyzer/pass_fail.go

```diff
@@ -481,22 +481,46 @@ func (a *weeklyAverageFromTenDays) CheckFailed(ctx context.Context, jobName stri
 	), nil
 }
 
-var testsRequiringHistoryRewrite = map[testCoordinates]string{
-	{
-		jobName:  "periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-upgrade",
-		testName: "[sig-api-machinery] OpenShift APIs remain available with reused connections",
-	}: "history correction on improperly loose disruption criteria, expires 2022-01-15",
-	{
-		jobName:  "periodic-ci-openshift-release-master-nightly-4.10-e2e-aws-upgrade",
-		testName: "[sig-api-machinery] Kubernetes APIs remain available with reused connections",
-	}: "history correction on improperly loose disruption criteria, expires 2022-01-15",
-}
+var testsRequiringHistoryRewrite map[testCoordinates]string
 
 type testCoordinates struct {
 	jobName  string
 	testName string
 }
 
+func init() {
+
+	testsRequiringHistoryRewrite[testCoordinates{
```

Comment:
panics on nil map.

Reply:
bah, sorry about that -- I didn't get to unit test it yet.

Reply:
fixed